online-shopping-website/src/Components/ProductPage.js

```diff
@@ -45,7 +45,7 @@ export const ProductPage = (props) => {
   return (
     <Grid container columnSpacing={4} rowSpacing={5}>
       <Grid item xs={12} md={3} lg={2}>
-        <SideNav brands={Brands} sellers={Sellers} onCheckboxChange={onCheckboxChange} onSliderChange={onSliderChange} />
+        <SideNav brands={Brands} sellers={Sellers} filterProducts={filterProducts} filters={filters} onCheckboxChange={onCheckboxChange} onSliderChange={onSliderChange} />
```

Comment:
For `Brands` and `Sellers` imports, I think we can just import them once from `TestValues.json` under the name `TestData` for brands, sellers, and products, since right now we have the same imported array being brought in but under three different names.